_problems/channel-jamming.md

```diff
@@ -0,0 +1,194 @@
+Jamming may also be classified into _capacity-based_ and _slot-based_.
+Capacity-based jamming is based on the fact that a payment of `X` coins locks `X` coins in-flight in every channel along the route.
+By sending a high-value jam, the attacker depletes the capacity of victim channels.
+Slot-based jamming exploits another limited resource, namely, payment slots.
+Each channel has up to 483 slots for concurrent in-flight payments.
+The attacker initiates many jams that occupy all available payment slots.
+Jams have lower value compared to capacity-based jamming, making slot-based jamming capital-efficient.
+
+## Impact
+
+Preventing jamming would improve Lightning reliability and facilitate its widespread adoption.
+If jamming is not efficiently prevented, routing nodes could request identification from their peers, diminishing Lightning's privacy guarantees.
+
+
+## Potential Directions
+
+Preventing jamming is non-trivial for two main reasons.
+First, Lightning payments are onion-routed.
+Intermediary nodes only know the immediate previous and next node in the route, but not the ultimate sender or receiver.
+This means that routing nodes cannot easily rate-limit or ban jammers.
+Second, failed payment attempts are free.
+The attacker only bears the cost of temporary capital lock-up (which is low for slot-based jamming).
+
+Proposed anti-jamming countermeasures include reputation systems and fees for failed payments.
+Both approaches have benefits and drawbacks.
+Reputation schemes weaken privacy guarantees, whereas extra fees introduce new game-theoretical and UX challenges.
+A practical solution could combine the two approaches.
+
+
+## Proposed Solutions
+
+### Node configuration
+
+Restricting payment forwarding may deter jamming to some extent.
+Node configuration doesn't require changes in the Lightning protocol.
+Configuration changes may include:
+
+1. Decrease payment timeouts. The attacker would have to send more jams per unit time to keep a channel jammed (for controlled jamming). The drawback is a shorter time window to claim in-flight payments after channel closure.
+2. Increase the minimal payment amount. The attacker would need more capital for slot-based jamming. The drawback is the loss of potential revenue from forwarding honest low-value payments.
+3. Divide payment slots into groups for different amount buckets. A routing node would forward small payments through a subset of slots, always keeping some slots open for larger payments. This increases the cost of controlled slot-based jamming. The drawback, again, is lost revenue: some honest payments may be rejected if their slot bucket is full, even if other slots are available.
+4. Limit the number of in-flight payments from less trustworthy peers. This may hinder attacks coming from an immediate peer but may also affect honest peers that are unknowingly forwarding jams.
+
+Configuration changes discourage simplistic attacks but don't solve the issue fundamentally.
```

Comment:
Since it's not a solution perhaps it could under a heading `Practical Mitigations` in the description.

Reply:
> under a heading Practical Mitigations in the description

Do you see `Practical Mitigation` as another first-level (`##`) heading?